model/import/SampleTemplateDownload.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * DownloadSampleCsv Controller provide actions to download sample CSV
+ *
+ * @author Chinnu Francis - TAO Team - {@link http://www.tao.lu}
+ * @package taoQtiItem
+ * @license GPLv2  http://www.opensource.org/licenses/gpl-2.0.php
+ */
```

Comment:
I feel that format differs from the default one; could you use like here https://github.com/oat-sa/extension-tao-itemqti/pull/1751/commits/a5d20579a5fffe53f86f4edde62ee7f725f09502#diff-4197dbe0e958d824be0e244e0b15db46a26868068b05291b7e885e48c137bcb6R4-R19

Reply:
I have done this